src/remote_connector_dao/GraphWidget2D.py

```diff
@@ -1,17 +1,13 @@
 import pyqtgraph as pg
+
 from remote_connector_dao.app_styles import (
-    colors,
+    Colors,
     graph_line_style,
     axis_label_style,
     graph_title_style,
 )
 
-LINE_COLORS = [
-    colors["red_primary"],
-    colors["blue_primary"],
-    colors["green_primary"],
-    colors["black"],
-]
+LINE_COLORS = [color.value for color in Colors if color.value != "#FFFFFF"]
```

Comment:
let's use the power of the abstraction
```suggestion
LINE_COLORS = [color.value for color in Colors if color != Colors.WHITE_PRIMARY]
```